cpp/log/ct_extensions.h

```diff
@@ -22,6 +37,8 @@ extern const char kEmbeddedSCTListOID[];
 extern const char kPoisonOID[];
 // Extended Key Usage value for Precertificate signing
 extern const char kPrecertificateSigningOID[];
+// Name constrained intermediate CA may not be logged
+extern const char kNameConstraintNologIntermediateOID[];
```

Comment:
I agree, it's fine to do it as a separate effort (and keep it consistent in this PR), I was just wondering if there was some reference that I missed.